support-frontend/test/selenium/conf/selenium-test.conf

```diff
@@ -9,6 +9,11 @@ identity {
   webapp.url = ${?IDENTITY_WEBAPP_URL}
   test.users.secret = ${?IDENTITY_TEST_USERS_SECRET}
 }
+idapi {
+  newTestUser.url = ${?IDAPI_NEW_TEST_USER_URL}
+  clientAccessToken.name = ${?IDAPI_CLIENT_ACCESS_TOKEN_NAME}
+  clientAccessToken.secret = ${?IDAPI_CLIENT_ACCESS_TOKEN_SECRET}
+}
```

Comment:
not something you've added, but it's funny that this whole file seems to be just an indirection from lower case dotted format to caps with underscores.

I'd be tempted to hard code the test user url in the code as it's not a secret and it's known at compile time (at least per stage) so that would simplify the amount of steps to get to the real value.

Reply:
Could do! I'd leave it as it is for now as this way all the IDAPI endpoint config values are stored in the same place in Travis, rather than being scattered in different places.